CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.5.3
+
+### Added
```

Comment:
I'd keep the same style we were using so I'd avoid sub-titles and just go with a plan list here.

Reply:
I was trying to follow Keep A Changelog since that's what we use for other projects. I also find the subheads more readable than having them inline. But I won't block on that, so let me know if I should still change it :)